triciaswift/tastevine-client
# Added Recipe Details view

## To Test
1. Login in to app
2. Go to either All Recipes or My Recipes
3. Click on one of the recipe cards
4. A page should display with a single recipe card with the tabs Recipe & Image
5. When the page is loaded recipe should automatically be active and you should see the following details
    - title
    - ingredients
    - directions
    - author
    - date
    - categories
6. If it is the current user's recipe then at the bottom under categories you should see a gear and a trash can icon

cloeses #12
